pkg/node/node.go

```diff
@@ -424,41 +429,109 @@ func (n *Node) publishSignature(
 			continue
 		}
 
+		if !n.confirmSignature(keepAddress, digest) {
+			time.Sleep(retryDelay) // TODO: #413 Replace with backoff.
+			continue
+		}
+
 		logger.Infof("signature submitted for keep [%s]", keepAddress.String())
```

Comment:
There is a little weird effect in the logs now:
```
2020-11-19T11:45:34.694+0100	INFO	keep-ecdsa	signature submission for keep [0xe175585397E143636920390ecfFc11828ec80E4e] has been confirmed
2020-11-19T11:45:34.694+0100	INFO	keep-ecdsa	signature submitted for keep [0xe175585397E143636920390ecfFc11828ec80E4e]
```

What do you think about removing this log, and letting `confirmSignature` print something like:
```
"signature for keep [%s] successfully submitted and confirmed"
```
?